Flaky test: test_large_data_will_fail_a_health_check

```
You can add @seed(338412295970389942002041176490929560425) to this test to reproduce this failure.
...
=================================== FAILURES ===================================
___________________ test_large_data_will_fail_a_health_check ___________________
[gw14] linux2 -- Python 2.7.15 /usr/bin/python2
Traceback (most recent call last):
  File "/build/python-hypothesis/src/hypothesis-3.68.0-py2/hypothesis-python/tests/cover/test_health_checks.py", line 127, in test_large_data_will_fail_a_health_check
    test()
  File "/usr/lib/python2.7/site-packages/_pytest/python_api.py", line 667, in __exit__
    fail(self.message)
  File "/usr/lib/python2.7/site-packages/_pytest/outcomes.py", line 97, in fail
    raise Failed(msg=msg, pytrace=pytrace)
Failed: DID NOT RAISE <class 'hypothesis.errors.FailedHealthCheck'>
===Flaky Test Report===
```

A simple retry passed it, so I assume it's a flaky test.

--- a/hypothesis-python/tests/cover/test_health_checks.py
+++ b/hypothesis-python/tests/cover/test_health_checks.py
@@ -119,7 +119,7 @@ def test_filtering_most_things_fails_a_health_check():
 
 
 def test_large_data_will_fail_a_health_check():
-    @given(st.lists(st.binary(min_size=1024)))
+    @given(st.none() | st.binary(min_size=1024))
     @settings(database=None, buffer_size=1000)
     def test(x):
         pass

